src/test/java/org/openrewrite/java/apache/httpclient5/AddTimeUnitArgumentTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.apache.httpclient5;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class AddTimeUnitArgumentTest implements RewriteTest {
+
+    @Test
+    void addTimeUnitDefaultMilliseconds() {
+        rewriteRun(
+          spec -> spec.recipe(new AddTimeUnitArgument("A method(int)", null)),
+          //language=java
+          java("""
+            import java.util.concurrent.TimeUnit;
+            
+            class A {
```

Comment:
It looks like we repeat this class three times; would it make sense to extract this to a constant? That helps focus the tests on the replacements and recipe in/output instead.